File-Formats/DocIO/Working-with-Table-Of-Contents.md

```diff
@@ -581,10 +581,356 @@ document.Close();
 //https://help.syncfusion.com/file-formats/docio/create-word-document-in-xamarin#helper-files-for-xamarin
 {% endhighlight %}
 
-{% endtabs %}  
+{% endtabs %}
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Table-Of-Contents/TOC-with-user-defined-styles).
 
+## Apply switches for table of contents
+
+You can modify the preservation of TOC entries in Word document by applying switches to table of contents. You can apply switches to preserve TOC entries as hyperlink or not, use default heading styles, preserve page numbers or not, aligning the page numbers, preserve new line characters, preserve outline level paragraphs, and preserve table of entry fields in the TOC entries using Word (DocIO) library.
+
+The following code example illustrates how to apply switches for table of contents.
+
+{% tabs %} 
+
+{% highlight c# tabtitle="C#" %}
+//Create a new Word document.
+using (WordDocument document = new WordDocument())
+{
+//Add a section into the Word document.
+IWSection section = document.AddSection();
+//Add a paragraph into the created section.
+IWParagraph paragraph = section.AddParagraph();
+//Append the TOC field with LowerHeadingLevel and UpperHeadingLevel to determines the TOC entries.
```

Comment:
Append the TOC field with LowerHeadingLevel and UpperHeadingLevel to determine the TOC entries.